packages/plugins/connection-manager/explorer/tree-items.ts

```diff
@@ -299,10 +299,15 @@ export class SidebarFunction extends SidebarAbstractItem<null> {
     this.tooltip = `${this.functionData.signature}(${this.functionData.args.join(',')}) => ${this.functionData.resultType || 'void'}`;
     this.value = `${this.functionData.signature}`;
     let args = [];
-    this.functionData.args.forEach((type, index) => {
-      args.push(`\${${index + 1}:${type}}`);
+    this.functionData.args.forEach((type, index) => {      
+      const [argName, argType] = type.split('=>');
+      if (argType !== null) {        
+        args.push(`${argName} => \${${index + 1}:${argType}}`);
+      } else {
+        args.push(`\${${index + 1}:${type}}`);
+      }      
     });
-    this.snippet = new SnippetString(`${this.functionData.signature}(${args.join(', ')})$0`);
+    this.snippet = new SnippetString(`${this.functionData.signature}(${args.join(', ')});$0`);
```

Comment:
I added `;` separator add the end. on the other hand if you use your function inside a select statement this is not neceseary. nevertheless func and proc are mostly used in blocks in my opinion.

Reply:
That's not 100% true. we can use procedures/functions inside of any part of the queries.

This is a bit overthinking, but it's better to add the ; when need, instead of removing, because the cursor will move to the end anyway

Reply:
Ok, I agree.